vitrina/compat/views.py

```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponseNotFound, HttpResponse
+from vitrina.compat.models import Redirections
+
+
+def redirection_handler(request, exception):
+    try:
+        existing_url = Redirections.objects.get(path=str(request.path))
+        new_url = existing_url.content_object
+        response = HttpResponse(status=308)
+        response['location'] = new_url
+        return response
+    except:
+        return HttpResponseNotFound()
```

Comment:
```suggestion
        return page_not_found(request, exception)
```

Please, leave default behavior untouched.

https://docs.djangoproject.com/en/3.2/ref/views/#django.views.defaults.page_not_found